mindexer/utils/query.py

```diff
@@ -161,6 +161,25 @@ def index_intersect(self, index):
 
         return query
 
+    def index_number_key_qurey(self, index):
```

Comment:
Typo in the function name. Can you change it here and where this is called? 

```suggestion
    def index_number_key_query(self, index):
```